inspirehep/modules/records/tasks.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Records tasks."""
+
+from __future__ import absolute_import, division, print_function
+
+from celery import shared_task
+from elasticsearch.helpers import scan
+from flask import current_app
+from six import iteritems
+
+from invenio_db import db
+from invenio_records.api import Record
+from invenio_search import current_search_client as es
+
+from inspirehep.modules.pidstore.utils import (
+    get_endpoint_from_schema,
+    get_index_from_endpoint,
+)
+
+
+@shared_task(ignore_result=True)
+def update_refs(old, new):
```

Comment:
What is `old` and `new`? Better naming of parameters and docstring would be useful.